src/Editor/json/tools_attr.json

```diff
@@ -0,0 +1,518 @@
+[
+  {
+    "name" : "[one-build]",
+    "body": [],
+    "description": "command line tool to run ONE drivers in customized order"
+  },
+  {
+    "name" : "[onecc]",
+    "body": [],
+    "description": "Run ONE driver via several commands or configuration file"
+  },
+	{
```

Comment:
These two have mis alignement.

Reply:
I think you used TAB key. Please replace with space character.

Reply:
Okay. I revised it!